
 This patch can be applied to version 3.0.1 and fixes a 
 syntax error that causes builds on older Solarises to fail.

 Apply it to version 3.0.1 with:

 cd <bacula-source>
 patch -p0 <3.0.1-solaris-acl.patch
 ./configure <your-options>
 make
 ...
 make install



Index: src/filed/acl.c
===================================================================
--- src/filed/acl.c	(revision 8810)
+++ src/filed/acl.c	(working copy)
@@ -1141,7 +1141,7 @@
 /*
  * OS specific functions for handling different types of acl streams.
  */
-static bool solaris_build_acl_streams(JCR *jcr, FF_PKT *ff_pkt);
+static bool solaris_build_acl_streams(JCR *jcr, FF_PKT *ff_pkt)
 {
    int n;
    aclent_t *acls;
